backend/pkg/logger/data/logger.go

```diff
@@ -0,0 +1,115 @@
+package data
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+	"sync"
+	"time"
+
+	"github.com/HyperloopUPV-H8/h9-backend/pkg/abstraction"
+	"github.com/HyperloopUPV-H8/h9-backend/pkg/logger"
+	"github.com/HyperloopUPV-H8/h9-backend/pkg/transport/packet/data"
+)
+
+type ValueName string
+
+const (
+	name abstraction.LoggerName = "data"
```

Comment:
```suggestion
	Name abstraction.LoggerName = "data"
```
so that other packages can reference this constant